docs: add main README and updated documentation usages (#2169)
scaleway/scaleway-ui: README.md

![](.storybook/assets/logo-dark.svg#gh-dark-mode-only)
![](.storybook/assets/logo-light.svg#gh-light-mode-only)

![Codecov](https://img.shields.io/codecov/c/github/scaleway/scaleway-ui)
![GitHub last commit](https://img.shields.io/github/last-commit/scaleway/scaleway-ui)
![GitHub closed issues](https://img.shields.io/github/issues-closed/scaleway/scaleway-ui)
![GitHub contributors](https://img.shields.io/github/contributors/scaleway/scaleway-ui)
![GitHub commit activity](https://img.shields.io/github/commit-activity/m/scaleway/scaleway-ui)
![GitHub](https://img.shields.io/github/license/scaleway/scaleway-ui)

# Scaleway UI Core

Scaleway UI Core contains the core features of the Scaleway UI library. 

> It is set of React library that can be used to build fast application.

Reviewer: ```suggestion
It's a set of React libraries that can be used to build fast application.
```
